packages/pipeline/src/Pipeline.story.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { array, select } from '@storybook/addon-knobs';
+import { css } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+import Pipeline, { Stage, Size, Variant } from '.';
+
+const containerStyle = css`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  overflow: hidden;
+  resize: horizontal;
+  padding: 2rem;
+  margin: 2rem;
+  border: 1px solid ${uiColors.gray.light1};
+  min-width: 180px;
+  width: 320px;
+`;
+
+function DefaultExample() {
+  // Configure knobs
+  const defaultStages = [
+    '$match',
+    '$group',
+    '$project',
+    '$addFields',
+    '$limit',
+  ];
+
+  const stages = array('Pipeline Stages', defaultStages).filter(Boolean);
+  const size = select('Size', Object.values(Size) as Array<Size>, Size.XSmall);
+  const variant = select(
+    'Variant',
+    Object.values(Variant) as Array<Variant>,
+    Variant.Default,
+  );
+
+  return (
+    <div className={containerStyle}>
+      <Pipeline size={size} variant={variant}>
+        {stages.map((stage, index) => (
+          <Stage key={`${stage}-${index}`}>{stage}</Stage>
```

Comment:
We prefer wrapping in a Fragment rather than using `key` props.

Reply:
This won't work due to [this code](https://github.com/mongodb/leafygreen-ui/blob/e8b91b5178f4406066a39edbd3c3a4e96b120df3/packages/pipeline/src/Pipeline.tsx#L206-L217) which transforms the child nodes into `Stage` components.


Essentially the Fragments will be converted to `Stage` components themselves, and if thet contain `Stage` components, the rendering will be incorrect.

Given the `key` prop is to simply prevent React warnings in this context, I am of the opinion that this is not a problem.